doc/userguide/upgrade/8.0-dns-logging-changes.rst

```diff
@@ -0,0 +1,260 @@
+:orphan: Document not referenced in a toctree, so add this.
```

Comment:
why is this not added to a toctree?

Reply:
> why is this not added to a toctree?

Its linked from the upgrade notes, like the unified2 stuff.  We don't yet have an "index" page for these, but the upgrade bullets kind of acts like that.

Reply:
ok, thanks